Core/Frameworks/BaikalAdmin/WWWRoot/install/index.php

```diff
@@ -25,7 +25,6 @@
 ***************************************************************/
 
 ini_set("session.cookie_httponly", 1);
-ini_set("display_errors", 0);
```

Comment:
Yea I want people to see errors, and if they don't like it, they should imho centrally set display_errors to 1. I have a feeling people will run into issues that I haven't foreseen, so we need them to be able to report bugs.